features/support/env.rb

```diff
@@ -6,6 +6,7 @@
 # instead of editing this one. Cucumber will automatically load all features/**/*.rb
 # files.
 
+ENV['RAILS_ENV'] = 'test'
```

Comment:
how about `||=`

Reply:
it explicitly sets rails env to 'test' instead of default 'development'